src/data/h1/tags/weapon.yml

```diff
@@ -454,34 +484,48 @@ comments:
               en: ...
         - name: error
           en: >
-            The function of this field is unknown. It is **possibly unused by the game**.
+            The function of this field is unknown. It is **unused by the game**.
             Although labeled in [Guerilla][] as a range \[0, 1\], the pistol has a max
             value of 2. Setting this to 0 does not affect projectile spread.
         - name: error acceleration time
           en: >
             Affects how long it takes for the weapon to reach its maximum projectile
-            error angle during rapid or automatic fire.
+            error angle while holding the trigger. Can be negated using quick
+            trigger taps.
         - name: error deceleration time
           en: >
             Affects how long it takes for the weapon to reach its minimum projectile
             error angle during a pause in firing.
         - name: charging time
-          en: ...
+          en: >
+            The amount of time the user has to hold the trigger before the weapon
+            is considered charged.
         - name: charged time
-          en: ...
+          en: >
+            The amount of time it takes _after_ the weapon becomes charged before
+            the overcharged behavior takes effect. **Unused if overcharged action
+            is set to none.**
         - name: overcharged action
-          en: ...
+          en: >
+            The reaction the weapon will have to the charged time running out.
           options:
             - name: none
-              en: ...
+              en: >
+                Nothing happens and the gun will stay charged until the trigger
+                is released.
             - name: explode
-              en: ...
+              en: >
+                The weapon will detonate using its [item][] _detonation effect_.
```

Comment:
You could link right to that field with `[item _detonation effect_][item#tag-field-detonation-effect]`.